Configure

```diff
@@ -882,7 +890,7 @@ while (@argvcopy)
                         {
                         push @{$useradd{LDLIBS}}, $_, shift(@argvcopy);
                         }
-                elsif (/^-L(.*)$/ or /^-Wl,/)
+                elsif (m|^[-/]L(.*)$| or /^-Wl,/)
```

Comment:
This one is meaningless with MSVC.  It's doubly meaningless because we use import libraries there, so are using direct library paths rather than having a search list.

Reply:
I just followed your suggestion https://github.com/openssl/openssl/pull/9961#discussion_r326856137 ;-).  But never mind. 

So if I understand you correctly I shall revert this line, correct?

Reply:
Yes, please.
We can probably extend it with knowledge of more linker flags of interest, but quite frankly, I think the variable passing method is better, it puts control over what goes where back in the hands of the users.

Reply:
Reverted in c71be2bc0c18c55e9d6c757a5ec7d23e65b66835.